.github/workflows/image.yaml

```diff
@@ -0,0 +1,79 @@
+name: release
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+jobs:
+  hnc-manager:
```

Comment:
I don't see anything here to build the manifests or plugins. Is the goal to add those later? They don't need to be in the first PR.

Reply:
Yes, will migrate the rest as separate PR.